Utils/chessCommands.py

```
@@ -7,15 +7,15 @@
 def getRandomOpening():
     # https://stackoverflow.com/a/3540346
     # Get random opening in list of openings
-    lines = open('Chess Openings.txt').read().splitlines()  # list
+    lines = open('../Data/Chess Openings.txt').read().splitlines()  # list
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)